persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -16,8 +16,15 @@ CREATE TABLE Conversation (
     muted_status TEXT AS ConversationEntity.MutedStatus DEFAULT "ALL_ALLOWED" NOT NULL,
     muted_time INTEGER DEFAULT 0 NOT NULL,
     creator_id TEXT NOT NULL,
+    -- when even a conversation event is recived, last_modified_date is updated for spicific events
+    -- e.g. updated when a new message text is received but not when a message is deleted
+    -- this is used for sorting conversations
     last_modified_date INTEGER AS Instant NOT NULL,
+    -- whenever a user is notified about the message belonging to this conversation
+    -- last_notified_date is updated, this would be typically a notification on the client side
     last_notified_date INTEGER AS Instant,
+    -- whenever a user receives a new message, last_read_date is being updated,
+    -- so that it reflects that timestamp of most fresh received message
```

Comment:
```suggestion
    -- whenever a user opens a conversation, last_read_date is updated with date of the last_modified_date,
    -- so that it reflects that timestamp of the last read message.
```